packages/api/tests/controllers/nomination.test.js

```diff
@@ -15,20 +17,38 @@ describe('GET Nomination Endpoint', () => {
   });
 
   it('returns a 200 when nomination exist', async () => {
-    const res = await request(app)
-      .get(`/nomination/${nomination.id}`);
+    const res = await request(app).get(`/nomination/${nomination.id}`);
     expect(res.statusCode).toBe(200);
   });
 
   it('return a 404 when nomination does not exist', async () => {
-    const res = await request(app)
-      .get('/nomination/00000000-0000-0000-0000-000000000000');
+    const res = await request(app).get(
+      '/nomination/00000000-0000-0000-0000-000000000000'
+    );
     expect(res.statusCode).toBe(404);
   });
 
   it('return a 400 when uuid is not a valid uuid', async () => {
-    const res = await request(app)
-      .get('/nomination/326');
+    const res = await request(app).get('/nomination/326');
     expect(res.statusCode).toBe(400);
   });
 });
+
+describe('Create Nomination Endpoint', () => {
+  it('returns 500 when field is null', async () => {
+    const data = {
+      providerFirstName: 'test provider',
+    };
+    const res = await request(app).post('/nomination').send(data);
+    expect(res.statusCode).toBe(500);
```

Comment:
I actually think that it makes more sense to throw a 400 (bad request) error in this case. The 500 error code is for situations where our app blew up to a mistake on our side.